superset-frontend/.eslintrc.js

```diff
@@ -195,9 +232,27 @@ module.exports = {
           'error',
           { devDependencies: true },
         ],
-        'jest/consistent-test-it': 'error',
         'no-only-tests/no-only-tests': 'error',
+        'max-classes-per-file': 0,
         '@typescript-eslint/no-non-null-assertion': 0,
+        'jest/consistent-test-it': 'error',
+        'jest/expect-expect': 0,
+        'jest/no-test-prefixes': 0,
+        'jest/valid-expect-in-promise': 0,
+        'jest/valid-expect': 0,
+        'jest/valid-title': 0,
+        'jest-dom/prefer-to-have-attribute': 0,
+        'jest-dom/prefer-to-have-text-content': 0,
+        'jest-dom/prefer-to-have-style': 0,
```

Comment:
Are these ignores really needed? If this is only temporary we should probably add a todo or preferably to fix any violating tests. Especially `jest/valid-expect-in-promise` feels like a pretty bad violation, and these should probably be fixed to avoid flaky tests.

Reply:
It looks ESlint change some default rule when I upgrade the ESlint. I will add a todo comment in these rules.
<img width="1912" alt="image" src="https://user-images.githubusercontent.com/2016594/141958361-a2bfbc09-a294-4ef2-9a00-fba8710c8aed.png">